.rhiza/tests/conftest.py

```
@@ -32,3 +43,30 @@ def logger():
+    Shared rather than per-module because each language layer asserts the same thing
+    against a different file — ``[project].version``, ``[package].version``, or Go's
+    ``Version`` constant — and because every layer's release config derives its current
+    version from this tag.
+
+    Args:
+        root: Repository root, from the ``root`` fixture.
+
+    Returns:
+        str: The highest version tag, e.g. ``v1.3.1``.
+    """
+    result = subprocess.run(  # nosec B603
```

Ruff (0.16.1)

[error] `subprocess` call: check for execution of untrusted input

(S603)